src/plugins/mer/merhardwaredevicewizardpages.h

```diff
@@ -67,11 +67,15 @@ private slots:
     static ProjectExplorer::Abi::Architecture detectArchitecture(
             const QSsh::SshConnectionParameters &sshParams, bool *ok);
     static QString detectDeviceName(const QSsh::SshConnectionParameters &sshParams, bool *ok);
+    static bool detectSdkClientTools(const QSsh::SshConnectionParameters &sshParams, bool *ok);
```

Comment:
Since this method is only called from a non-static method, it doesn't have to be static. And since the return value is used for setting m_sdkClientToolsInstalled, I would even go so far that I would manipulate m_sdkClientToolsInstalled and m_connectionTestOk directly in detectSdkClientTools().

Reply:
... I can see that you are following the example set by detectDeviceName here, so may be it's not worth it changing the implementation here. Consider the comment above more like just food for thought than a change request.